### PR TITLE
Blog post on puppet-os-hardening 2.0.0

### PR DESCRIPTION
**Pull request by mcgege**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,143 +0,0 @@
-GEM
```
Comment (artem-sidorenko) [inline Gemfile.lock:1]: @chris-rock any idea why Gemfile.lock was commited in this repo? Do we need it this way?

**Review by artem-sidorenko — APPROVED** [review]

**Comment by chris-rock** [comment]
No I do not know

**Comment by mcgege** [comment]
I've always seen this file excluded till now and it was rebuilt when I run Jekyll locally, so I thought it was included by mistake. But if you're unsure I can change my PR ...

**Comment by chris-rock** [comment]
@mcgege lets remove this in a separate PR

**Review by chris-rock — APPROVED** [review]
Awesome work @mcgege

**Pull request merged**